lib/service/cfg.go

```diff
@@ -425,6 +425,10 @@ type ProxyConfig struct {
 	// advertises for Mongo clients.
 	MongoPublicAddrs []utils.NetAddr
 
+	// PeerPublicAddrs is the public addresse the proxy advertises for proxy
+	// peering clients.
```

Comment:
```suggestion
	// PeerPublicAddrs is the public address the proxy advertises for proxy
	// peering clients.
	// Defaults to PeerAddr.
```

I'd also move it below `PeerAddr`, since the fields are related.